department-of-veterans-affairs/abd-vro
# Update logs to fix misaligned coverage stats in datadog

## What was the problem?

Logs that are for CfI will sometimes log "lookup table match", so we don't have an accurate message to determine NEW vs CfI claims in datadog.

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/361

## How does this fix it?[^1]

- this adds a new logging statement per classifier request in order to provide a single log message to draw datadog graphs from

## How to test this PR
- `pytest`
- running api locally


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
